app/views/devise/registrations/new.html.erb

```diff
@@ -4,10 +4,11 @@
         <%= simple_format t 'global.welcome', organization_name: t('global.organization_name') %>
       </h1>
       <div class="p-2 w-md-75">
-        <%= bootstrap_form_for resource, as: resource_name, url: registration_path(resource_name) do |f| %>
+        <%= bootstrap_form_for resource, as: resource_name, url: user_registration_path do |f| %>
           <h3 class="font-weight-light mb-1"><%= t '.sign_up' %></h3>
           <h4 class="font-weight-lighter"><%= t '.sign_up_prompt' %></h4>
           <hr />
+          <%= f.text_field :locale, value: I18n.locale, hidden: true, hide_label: true %>
```

Comment:
I thought we talked about not putting this on the sign-up page due to it probably causing clutter. Would have to take a look at it in the view. I thought this would be part of `app/views/devise/registrations/edit.html.erb`.